theseus/optimizer/nonlinear/tests/common.py

```diff
@@ -110,7 +110,7 @@ def _check_nonlinear_least_squares_fit(
     objective.update(values)
     initial_error = objective.error_squared_norm() / 2
     max_iterations = 20
-    optimizer = nonlinear_optim_cls(objective)
+    optimizer = nonlinear_optim_cls(objective, vectorize=False)
```

Comment:
> old stand-alone optimizer tests have `vectorize=False` because the copy method of the CostFunction used is not implemented

This sounds fine. Maybe leave a comment in the code for this.